docs/src/main/asciidoc/JavaEE_Tutorial.adoc

```diff
@@ -28,7 +28,7 @@ Coming Soon
 This guide is still under development, check back soon!
 ====
 
-include::_javaee-guide/Java_API_for_RESTful_Web_Services_(JAX-RS).adoc[]
+include::_javaee-guide/Jakarta_RESTful_Web_Services.adoc[]
```

Comment:
JAX-RS appears in Jakarta_RESTful_Web_Services_Reference_Guide.adoc three times.

Also, "defined in http://jcp.org/en/jsr/detail?id=311[JSR331])" is out of date.

"JBossAS 7" occurs seven times.

Reply:
following this document - https://jakarta.ee/legal/acronym_guidelines/